tests/images.rs

```diff
@@ -223,6 +223,39 @@ fn generic_image() {
     assert_eq!(first_column, 2);
 }
 
+#[test]
+fn generic_image_with_custom_entrypoint() {
+    let docker = clients::Cli::default();
+    let msg = images::generic::WaitFor::message_on_stdout("server is ready");
+
+    let generic = images::generic::GenericImage::new("tumdum/simple_web_server:latest")
+        .with_wait_for(msg.clone());
+
+    let node = docker.run(generic);
+    let port = node.get_host_port(80).unwrap();
+    assert_eq!(
+        "foo",
+        reqwest::blocking::get(&format!("http://127.0.0.1:{}", port))
+            .unwrap()
+            .text()
+            .unwrap()
+    );
+
+    let generic = images::generic::GenericImage::new("tumdum/simple_web_server:latest")
+        .with_wait_for(msg)
+        .with_entrypoint(&"/bar");
```

Comment:
Having to pass a reference to a string slice is a bit odd. That is effectively a double reference. Any chance we can fix that?

Reply:
That reference wasn't needed.